assemble/conf/templates/accumulo-env.sh

```diff
@@ -48,7 +48,8 @@ JAVA_OPTS=("${ACCUMULO_JAVA_OPTS[@]}"
   '-XX:OnOutOfMemoryError=kill -9 %p'
   '-XX:-OmitStackTraceInFastThrow'
   '-Djava.net.preferIPv4Stack=true'
-  "-Daccumulo.native.lib.path=${lib}/native")
+  "-Daccumulo.native.lib.path=${lib}/native"
+  "-Daccumulo.dynamic.classpaths=${lib}/ext/[^.].*.jar")
```

Comment:
Do we even need this property? Shouldn't the default value simply be blank in the site file?